src/context.ts

```diff
@@ -492,7 +492,7 @@ class Context extends EventEmitter {
   }
 
   _urlForPath(path: string, isWebSocket: boolean, options: MojoURLOptions): string {
-    path = this.req.basePath + path;
+    path = this.req.basePath + (path || '/');
```

Comment:
Considering everything that can be `false` in JavaScript, that looks prone to unintended side effects. 🤔

Reply:
Sure, done